components/responsive/responsive.js

```diff
@@ -0,0 +1,146 @@
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import sizes from 'travix-breakpoints/sizes';
+import debounce from 'lodash.debounce';
```

Comment:
better do `lodash/debounce`. will play well along with our `externals` configuration in Webpack.

Reply:
oke